internal/pkg/template/templates/environment/cf.yml

```
@@ -278,6 +363,15 @@ Resources:
     Condition: CreateALB
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
     Properties:
+      {{- if .PublicHTTPConfig.ELBAccessLogs }}
+      LoadBalancerAttributes:
+        - Key: 'access_logs.s3.enabled'
+          Value: true
+        - Key: 'access_logs.s3.prefix'
+          Value: {{ .PublicHTTPConfig.ELBAccessLogs.BucketPrefix }}
```

Should this be:
```suggestion
        {{- if  .PublicHTTPConfig.ELBAccessLogs.BucketPrefix }}
        - Key: 'access_logs.s3.prefix'
          Value: {{ .PublicHTTPConfig.ELBAccessLogs.BucketPrefix }}
        {{- end}}
```
So that a prefix is there only if it's not empty 💭

---

So prefix will be there in either case as we are setting it to default even if customers doesn't provide it, similar to how if customers doesn't provide bucket name but only provides prefix then we create the bucket and attach their prefix to it.

---

Can we remove our default prefix plz 🥺 I feel like the console experience makes sense because they place logs already under `/AWSLogs/aws-account-id/elasticloadbalancing/` which is pretty clear enough